### PR TITLE
Rooms page design

### PR DESCRIPTION
**Pull request by ahzamir**
### In this pull request I have added some more designs to my Room component:

- I added the carousel slider to the Rooms component items list.
- I added some designs to my slider and items.
- I added some designs to my social account icons.
- I added onClick functions for social icons to share the Room on social media accounts.

### CODE REVIEW

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
## Required Changes ♻️
## Hi @ahzamir , 
 ### You have been doing a tremendous job so far :ok_hand: We are almost there!

## To Highlight 🎉
- PR has a title and summary. :heavy_check_mark:
- Separated CSS file :heavy_check_mark:
- Slick design :heavy_check_mark:
## To Improve 🙏
- [ ] According to the design provided, we have no borders. We need to stick to the[ design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign) as much as possible and removing the borders will not affect our design.

Screen Shot 2022-10-06 at 6 45 39 PM


## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/Rooms/Rooms.js [hunk 1]
```diff
@@ -3,40 +3,161 @@ import { useSelector, useDispatch } from 'react-redux';
...
+        customRightArrow={<CustomRightArrow />}
+        customLeftArrow={<CustomLeftArrow />}
+        removeArrowOnDeviceType={['tablet', 'mobile']}
+      >
         {rooms.map((room) => (
-          <a href={`/room/${room.id}`} key={room.id} className="col text-decoration-none text-dark">
-            <div className="card h-100 shadow border-0 rounded-3 bg-light text-center p-3 p-md-5 transition" key={room.id}>
-              <img src={room.image} alt={room.name} className="card-img-top rounded-3 mb-3 mb-md-4 mx-auto" style={{ width: '300px', height: '200px' }} />
+          <a href={`/room/${room.id}`} key={room.id} className="text-decoration-none text-dark room-card">
+            <div className="card h-100 shadow border-0 rounded-3 bg-light text-center p-3 p-md-5 transition room-card" key={room.id}>
```
Comment (huseyinbiyik) [inline src/components/Rooms/Rooms.js:127]: You can delete  'shadow' from this line.

File: src/components/Rooms/Rooms.js [hunk 2]
```diff
@@ -3,40 +3,161 @@ import { useSelector, useDispatch } from 'react-redux';
...
+    onClick: PropTypes.func.isRequired,
+  };
+
+  CustomLeftArrow.propTypes = {
+    onClick: PropTypes.func.isRequired,
+  };
+
   return (
     <div>
       <h1 className="text-center mt-5 mb-2 pt-5 fw-bold fs-1">Rooms</h1>
```
Comment (huseyinbiyik) [inline src/components/Rooms/Rooms.js:101]: Please change the 'Rooms' to uppercase according to the design.

**Comment by ahzamir** [comment]
@huseyinbiyik  Thank you very much for your kind review and feedback. all required changes have been implemented. 👍😊✅

**Review by huseyinbiyik — APPROVED** [review]
Hi @huseyinbiyik, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Changes implemented ✔️ 






Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by ahzamir** [comment]
@huseyinbiyik  Thank you very much for your kind review and feedback. ✅😊👍

**Pull request merged**